eng/Tools.props

```diff
@@ -1,6 +1,6 @@
 <Project>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
```

Comment:
❓ Why is **Tools.props** being included in sourcebuild? Can we exclude the file through some other means?

Reply:
It looks like it is included by Arcade: https://github.com/dotnet/arcade/blob/aa647dcfc48bd3a644625e549ff71b7d06ed0958/src/Microsoft.DotNet.Arcade.Sdk/tools/Tools.proj#L53